src/Mod/PartDesign/Gui/TaskPrimitiveParameters.ui

```diff
@@ -761,31 +989,40 @@
           </widget>
          </item>
          <item row="0" column="1">
-          <widget class="Gui::QuantitySpinBox" name="torusAngle3" native="true">
+          <widget class="Gui::QuantitySpinBox" name="torusAngle3">
+           <property name="keyboardTracking">
+            <bool>false</bool>
+           </property>
            <property name="unit" stdset="0">
             <string notr="true">deg</string>
            </property>
-           <property name="value" stdset="0">
+           <property name="value">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
-          <widget class="Gui::QuantitySpinBox" name="torusAngle1" native="true">
+          <widget class="Gui::QuantitySpinBox" name="torusAngle1">
+           <property name="keyboardTracking">
+            <bool>false</bool>
+           </property>
            <property name="unit" stdset="0">
             <string notr="true">deg</string>
            </property>
-           <property name="value" stdset="0">
-            <double>-180.000000000000000</double>
+           <property name="value">
+            <double>180.000000000000000</double>
```

Comment:
Do you have removed the minus on purpose? This is wrong because the range is [-180, 0]

Reply:
> Do you have removed the minus on purpose? This is wrong because the range is [-180, 0]

No. At first the values disappeared because of the bug in the designer plugin. Then you fixed the plugin and I try to restore the values. Obviously this did not work for the minus signs. I am sorry for not noticing it.